src/components/dls/Link/Link.tsx

```diff
@@ -23,7 +23,7 @@ type LinkProps = {
 const Link: React.FC<LinkProps> = ({ href, children, newTab = false, variant, download }) => (
   <Wrapper shouldWrap={!download} wrapper={(node) => <NextLink href={href}>{node}</NextLink>}>
     <a
-      href={download ? href : undefined}
```

Comment:
hmmm, interesting. Nextjs Link wants `href` inside the `a`. 
If this is the case maybe we can 
- just do `href={href}` directly 
- or use `passHref` in `NextLink` not sure if this one will works or not

Reply:
gonna do href={href} this is what makes sense to me as well